frontend/js/tests/test-utils/rtl-test-utils.tsx

```diff
@@ -52,7 +53,7 @@ export const renderWithProviders = (
     return (
       <GlobalAppContext.Provider value={globalProps}>
         <ToastContainer />
-        {children}
+        <BrowserRouter>{children}</BrowserRouter>
```

Comment:
Note to self: figured out how we need to layer these iup and in what order, see https://github.com/metabrainz/listenbrainz-server/blob/6085b8d9c0f96ae4e48ed657a0f785b9f02b3ea4/frontend/js/tests/test-utils/rtl-test-utils.tsx#L53-L59